### DISCUSSION TITLE
Region resize start handler

### DISCUSSION THREAD

**Discussion by Olejqua** [katspaugh/wavesurfer.js#3519] [Q&A] [answered]
Greetings, everyone!
I need to track when a user starts to resize a region.
Is there any way to track the mousedown event?

**Comment by katspaugh** [comment, marked as answer]
Regions have an `update` event, and it sends a `side` parameter which tells you from which side it's resized. If it's not passed, it's not resized but moved.
See the docs: https://wavesurfer.xyz/docs/types/plugins_regions.RegionEvents

**Reply by Olejqua** [reply]
Thank you!
